OpenSim/Simulation/Model/PrescribedForce.cpp

```diff
@@ -175,7 +188,8 @@ void PrescribedForce::setTorqueFunctionNames
     for(int i=0;i<aFunctionNames.getSize();i++)
     {
         kineticsStore.getDataColumn(aFunctionNames[i], column);
-        tSpline[i]= new SimmSpline((forceSize>10?10:forceSize), t, column, aFunctionNames[i]);
+        tSpline[i]= new SimmSpline((forceSize>10?10:forceSize), t, column,
```

Comment:
Changing `(forceSize>10?10:forceSize)` to `std::min(forceSize, 10)` would improve readability IMHO. Also appears on L214.